pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -183,3 +186,36 @@ def test_dataframe_constructor_with_different_dtype_raises():
     xpr = "Cannot coerce extension array to dtype 'int64'. "
     with tm.assert_raises_regex(ValueError, xpr):
         pd.DataFrame({"A": arr}, dtype='int64')
+
+
+_ts = pd.Series(DecimalArray(make_data()))
+
+
+class TestOperator(BaseDecimal, TestSeriesOperators):
+    @cache_readonly
```

Comment:
never do this. use a fixture

Reply:
This is unavoidable, because I am subclassing the `tests.series.tests_operators.TestSeriesOperators` class.  That class inherits from `tests.series.common.TestData`, which defines `ts` as a `@cache_readonly` property.  The method I use for testing (`TestSeriesOperators.test_operators()`) has references to `self.ts`, i.e., it doesn't use a fixture, and I can't see how to override that definition so we get a `DecimalArray` series and reuse all that testing code.

Reply:
Never mind, I redid the tests as you suggested elsewhere